### ISSUE TITLE
The result is not good as the sample show

### ISSUE THREAD

**Issue by TouShienn** [frederic-bousefsaf/ippg_luv_skindetect_cwt#2]
Hello, I use my own face picture which is segmented from my video to run the program ,howerver ,the result ippg signal is not good as the picture show below.My picture pixel is higher than your sample ,but I found your face pictures is more bright and your background is  
black,do you use some color enhancement methods or euler video magnificance method to improve your face pictures quality?I really want to know why my picture does not work,hope your response,thank you very much.
![测试](https://user-images.githubusercontent.com/87792784/149652896-8892191e-3c8d-4366-a014-99d09a2dcb9b.jpg)

**Comment by frederic-bousefsaf** [comment]
Dear TouShienn,
Hard for me to comment without data... The sample video I provided has been recorded with a standard camera (autoexposure disabled). Maybe you can try to display the signal computed with green pixels or implement CHROM or POS techniques and observe the improvement.
Sincerely

**Comment by TouShienn** [comment]
Thank you for response. If your sample video does not use any method to enhance, do you think maybe the problem is my video quality is too bad due to the environment or camera,because I just use my computer camera to get the video.

**Comment by frederic-bousefsaf** [comment]
Indeed, that could be a potential reason. Try to block auto-exposure and auto-whitebalance because these parameters can have a huge impact on the quality of the PPG signal constituted after the spatial averaging operation.
